src/Components/AddUser.js

```diff
@@ -0,0 +1,113 @@
+import React, {Component} from 'react';
+import Container from "@material-ui/core/Container";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import styled from 'styled-components';
+import {InputLabel} from "@material-ui/core";
+import FormGroup from "@material-ui/core/FormGroup";
+
+const ContainerStyled = styled(Container)`
+    flex-grow: 1;
+    background: #f5f5f5;
+`;
+
+const TextFieldStyled = styled(TextField)`
+    background: #FEFEFE;
+    margin :30px;
+    width: 80%;
+`;
+
+const InputLabelStyled = styled(InputLabel)`
+    text-align: left;
+    margin-left: 125px;
+`;
+
+const ContainerButton = styled.div`
+    display : flex;
+    justify-content: space-between;
+    margin-top:40px;
+
+`;
+
+class AddUser extends Component {
```

Comment:
Let's use react hooks everywhere. So change this component to use react hooks instead of classes.